GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -218,8 +219,37 @@ public virtual string DisplayText()
                 return ToString();
             }
 
+            protected void SetNodeFont(FontStyle style)
+            {
+                if (style == FontStyle.Regular)
+                {
+                    // For regular, set to null to use the NativeTreeView font
+                    if (TreeViewNode.NodeFont != null)
+                    {
+                        TreeViewNode.NodeFont.Dispose();
```

Comment:
Is the performance change due to this noticable at all?
(I have not tried to find out. I assume it is OK.)

Reply:
Performance should be okay since only one branch node every needs to be null, so in the worst case, when rebuilding nodes, it will dispose once, and allocate a new Font once.